curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-modern-javascript-methods-by-building-football-team-cards/63e042661ad2663f0d468740.md

```diff
@@ -15,7 +15,7 @@ Use object destructuring to access the `coachName` value.
 
 ```js
 
-assert.match(code, /const\s*{\s*coachName\s*}\s*=\s*myFavoriteFootballTeam\.headCoach\s*/)
+assert.match(code, /const\s*{\s*coachName\s*}\s*=\s*myFavoriteFootballTeam\.headCoach\s*|const\s*{\s*headCoach\s*:\s*{\s*coachName\s*}\s*}\s*=\s*myFavoriteFootballTeam(?!.headCoach)\s*/)
 
```

Comment:
```suggestion
```

remove these white spaces and update the regex so that semicolon can be added optionally. it can be done using `;?` at end.

Reply:
no need to remove these whitespaces semicolon is optional already, we don't need to add `;?`

I have tested it in my dev env, it is working fine